paasta_tools/metrics/metastatus_lib.py

```diff
@@ -679,7 +725,7 @@ def suffixed_number_value(s: str) -> int:
     number, suff = match.groups()
 
     if suff in _IEC_NUMBER_SUFFIXES:
-        return int(number) * _IEC_NUMBER_SUFFIXES[suff]
+        return int(int(number) * _IEC_NUMBER_SUFFIXES[suff])
```

Comment:
I think it is fine to use int here since the difference is negligible for the resource utilization calculation. I keep the default CPU value here so that it is consistent with memory and disk, but I don't expect any user will be aware of this.

Reply:
But since we convert each pod's CPU request to an int before summing them, the sum could be very far off. Imagine a 16-core box with 32 0.5-cpu pods on it -- we'd report the usage as 0% but the actual usage would be 100%.

Reply:
Hmm you're right. I will change it to float.